### PR TITLE
DOC: mention discourse main page

### PR DESCRIPTION
**Pull request by jklymak**
## PR Summary

Add a mention of discourse to the front page

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -119,12 +119,15 @@ Software Foundation Code of Conduct <coc_>`_ in everything we do.
 
 Check the :doc:`FAQ <faq/index>` and the :doc:`API <api/index>` docs.
 
-For help, join the `gitter channel`_ and the matplotlib-users_,
-matplotlib-devel_, and matplotlib-announce_ mailing lists, or check out the
-Matplotlib tag on stackoverflow_.  The `search <search.html>`_ tool searches
-all of the documentation, including full text search of over 350 complete
+For help, write us at `discourse.matplotlib.org`_.  If its something quick
+join the `gitter channel`_ to chat with us.  There are alsothe
```
Comment (tacaswell) [inline doc/index.rst]:
```suggestion
join the `gitter channel`_ to chat with us.  There are also the
```

Comment (jklymak) [reply doc/index.rst]: Ooops!

File: doc/index.rst [hunk 2]
```diff
@@ -119,12 +119,15 @@ Software Foundation Code of Conduct <coc_>`_ in everything we do.
 
 Check the :doc:`FAQ <faq/index>` and the :doc:`API <api/index>` docs.
 
-For help, join the `gitter channel`_ and the matplotlib-users_,
-matplotlib-devel_, and matplotlib-announce_ mailing lists, or check out the
-Matplotlib tag on stackoverflow_.  The `search <search.html>`_ tool searches
-all of the documentation, including full text search of over 350 complete
+For help, write us at `discourse.matplotlib.org`_.  If its something quick
```
Comment (story645) [inline doc/index.rst]: The discourse is for more than just help though? Maybe something like " Join our community at 'discourse' to get help, discuss contributing & development, and share your work?

Comment (jklymak) [reply doc/index.rst]: Did this plus a bit of re-formatting to bulleted list

**Comment by jklymak** [comment]
https://25695-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/index.html

**Pull request merged**